AwaMelvine/fast-food-fast
# Change the design of the footer 164977940

#### What does this PR do?
Changes the design of the footer.
#### Description of Task to be completed?
- Write the markup and styling for a new footer
- Change the font family of the headings and text in the footer

#### How should this be manually tested?
Visit any public page (except auth pages) and check if the footer design looks different (better) than the previous design.
#### What are the relevant pivotal tracker stories?
[164977940] Enhance the design of the footer
